sdk/keyvault/keyvault-admin/README.md

```diff
@@ -34,16 +33,160 @@ TypeScript users need to have Node type definitions installed:
+
+Client secret credential authentication is being used in this getting started section but you can find more ways to authenticate with [Azure identity][azure-identity]. To use the [DefaultAzureCredential][DAC] provider shown below, or other credential providers provided with the Azure SDK, you should install the [@azure/identity][identity-npm] package:
+
+```PowerShell
+npm install @azure/identity
+```
+
+#### Create/Get credentials
+
+Use the [Azure CLI][azure-cli] snippet below to create/get client secret credentials.
+
+ * Create a service principal and configure its access to Azure resources:
+    ```PowerShell
+    az ad sp create-for-rbac -n <your-application-name> --skip-assignment
+    ```
+    Output:
+    ```json
+    {
+        "appId": "generated-app-ID",
+        "displayName": "some-app-name",
+        "name": "http://some-app-name",
+        "password": "random-password",
+        "tenant": "tenant-ID"
+    }
+    ```
+* Take note of the service principal objectId
+    ```PowerShell
+    az ad sp show --id <appId> --query objectId
+    ```
+    Output:
+    ```
+    "<your-service-principal-object-id>"
+    ```
+* Use the returned credentials above to set  **AZURE_CLIENT_ID** (appId), **AZURE_CLIENT_SECRET** (password), and **AZURE_TENANT_ID** (tenant) 
+environment variables. For example, you can use the [dotenv][dotenv] project and add a `.env` file to your project with hte following contents (for the full section of environment variables, see [Environment Variables](#environment-variables)):
+    ```PowerShell
+    AZURE_TENANT_ID=<tenant-ID>
+    AZURE_CLIENT_ID=<generated-app-ID>
+    AZURE_CLIENT_SECRET=<random-password>
+    ```
+
+* Create the Key Vault and grant the above mentioned application authorization to perform administrative operations on the Azure Key Vault 
+(replace `<your-resource-group-name>` and `<your-key-vault-name>` with your own, unique names and `<your-service-principal-object-id>` with the value from above):
+    ```
+    az keyvault create --hsm-name <your-key-vault-name> --resource-group <your-resource-group-name> --administrators <your-service-principal-object-id> --location <your-azure-location>
+    ```
+
+* Use the above mentioned Azure Key Vault name to retrieve details of your Vault which also contains your Azure Key Vault URL:
+    ```PowerShell
+    az keyvault show --hsm-name <your-key-vault-name>
+    ```
+
+#### Get or create a storage account
+
+To generate Key Vault backups, you will need to point the [KeyVaultBackupClient][src-backup-client] to an existing Storage account.
+
+To create a new Storage Account, you can use the [Azure Portal][storage-account-create-portal],
+[Azure PowerShell][storage-account-create-ps], or the [Azure CLI][storage-account-create-cli].
+Here's an example using the Azure CLI:
+
+```Powershell
+az storage account create --name MyStorageAccount --resource-group MyResourceGroup --location westus --sku Standard_LRS
+```
+
+#### Environment Variables
```

Comment:
I am not convinced that having this section here on env variables, sample.env and dotenv package in the readme is useful for the getting started scenario. Those copy/pasting code from the readme file should be able to do the bare minimum to get things started.

I see the merit of this in the Examples section because we are linking to samples that expect many env variables to be set.

Reply:
I'll remove it, and I'll link to the samples readme better!

Reply:
Here's what I'm placing in the examples:

> ## Examples
> 
> We have provided readmes with the detailed steps to run our samples, both for [JavaScript](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-admin/samples/javascript/README.md) and [TypeScript](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-admin/samples/typescript/README.md). Direct links to the specific samples follow:
>
> ...